SPECS/libusb/libusb.spec

```diff
@@ -55,20 +59,36 @@ popd
 %{_libdir}/pkgconfig/*
 
 %changelog
+* Mon Aug 30 2021 Bala <balakumaran.kannan@microsoft.com> - 1.0.24-2
+- Merged to 1.0 from dev branch
```

Comment:
Similar comment as before. If nothing changed on top of Thomas' version, then an additional entry is not necessary.